src/frontend/screens/Onboarding/walletSelection/screens/import/components/BrowsersAndManualImportOptions/index.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react'
+import { BrowsersAndManualImportOptionsProps } from '../../types'
+import { BrowserPackageManagerImportOption } from '../BrowserPackageManagerImportOptions'
+import { Button } from '@hyperplay/ui'
+import styles from './index.module.scss'
+import { t } from 'i18next'
+
+export function BrowsersAndManualImportOptions({
+  importOptions,
+  handleImportMmExtensionClicked,
+  browserSelected,
+  setBrowserSelected
+}: BrowsersAndManualImportOptionsProps) {
+  const [showManualImport, setShowManualImport] = useState(false)
+  const [manualImportPath, setManualImportPath] = useState(
+    '<profile>/Local Extension Settings/<metamask app id>'
```

Comment:
Bit unsure on this, should we translate this or the purpose is different?

Reply:
it is the file path. It is hardcoded to the English string "Local Extension Settings" in chromium https://github.com/chromium/chromium/blob/db23f044404ad327600e4f657342297c01d14e3b/extensions/common/constants.h#L110

i'll add translations for profile and app id though